src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1007,35 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Sets the cursor ftux key in localStorage
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    setCursorFtux(key) {
+        this.cache.set(key, true, true);
+    }
+
+    /**
+     * Updates localStorage with the state of the ftux cursor toggle
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    applyCursorFtux(key) {
+        const keyToClassMap = {
+            [IMAGE_FTUX_CURSOR_SEEN_KEY]: CLASS_ANNOTATIONS_IMAGE_FTUX_CURSOR_SEEN,
+            [DOCUMENT_FTUX_CURSOR_SEEN_KEY]: CLASS_ANNOTATIONS_DOCUMENT_FTUX_CURSOR_SEEN,
+        };
+
+        if (this.containerEl) {
```

Comment:
Can each viewer (document, image) maintain its own version of this with its specific key and class name? There's not much logic left anymore to require a method in BaseViewer.